packages/traceability/issuer-api/src/pods/certificate/handlers/certificate-created.handler.ts

```diff
@@ -63,21 +69,47 @@ export class CertificateCreatedHandler implements IEventHandler<CertificateCreat
             });
         }
 
-        const certificate = await this.repository.save({
-            id: cert.id,
-            blockchain: blockchainProperties,
-            deviceId: cert.deviceId,
-            generationStartTime: cert.generationStartTime,
-            generationEndTime: cert.generationEndTime,
-            creationTime: cert.creationTime,
-            creationBlockHash: cert.creationBlockHash,
-            owners: cert.owners,
-            issuedPrivately: !!privateInfo,
-            latestCommitment
-        });
-
-        this.eventBus.publish(new CertificatePersistedEvent(certificate.id));
+        const txHash = cert.creationTransactionHash.toLowerCase();
+        const unminedCommitment = await this.checkCommitment(txHash);
+
+        const queryRunner = this.connection.createQueryRunner();
+        await queryRunner.connect();
+        await queryRunner.startTransaction();
```

Comment:
I don't see transaction `commit`. Or does this query runner doesn't work like that?

Reply:
You are right, there should be a commit. I guess it auto-assumes on release that a commit was wanted and does it automatically.

Will add it explicitly also too.